integration_tests/commands/resp/hsetnx_test.go

```diff
@@ -1,9 +1,9 @@
-package async
+package resp
 
 import (
 	"testing"
 
-	"github.com/stretchr/testify/assert"
+	"gotest.tools/v3/assert"
```

Comment:
why are we going back to assert package here?

Reply:
this was unintentional, reverted the changes to keep it consistent.